runtime/vm/jfr.cpp

```diff
@@ -719,6 +722,7 @@ initializeJFR(J9JavaVM *vm, BOOLEAN lateInit)
 
 	vm->jfrState.prevSysCPUTime.timestamp = -1;
 	vm->jfrState.prevProcTimestamp = -1;
+	vm->jfrState.prevContextSwitchTimestamp = -1;
```

Comment:
This should explicitly initialize `prevContextSwitches` (or have a comment explaining why that's not necessary).